client/src/controllers/RevealController.ts

```diff
@@ -29,10 +29,12 @@ export class RevealController extends Controller<HTMLElement> {
     closed: { default: false, type: Boolean },
     peeking: { default: false, type: Boolean },
     peekTarget: { default: '', type: String },
+    saveState: { default: false, type: Boolean },
```

Comment:
Can I suggest we use `store`?

`data-w-breadcrumbs-store-value="true"` - reads a bit better, is shorter and avoids potential confusion with saving data elsewhere vs using local storage.

Reply:
```suggestion
    store: { default: '', type: String },
```

Another suggestion, instead of using the element id, we can just make `storeValue` a string. This way it gives a bit more flexibility, we can store something across ALL instances of a controller or just some, irrespective of an ID.

I feel that reliance on the `id` boxes us in a bit for other usage in side panels and customised usage in the future where the id can sometimes be generated automatically (e.g. panels).

## Examples

### Do not store at all, set to empty or do not set value

```
<div data-controller="w-breadcrumbs" />
<div data-controller="w-breadcrumbs" data-w-breadcrumbs-store-value="" />
```

### Store as the 'header' key

```
<div data-controller="w-breadcrumbs" data-w-breadcrumbs-store-value="header" />
```

### Store as the key across multiple, will be saved together

```
<div data-controller="w-reveal" data-w-reveal-store-value="grouping" />
<div data-controller="w-reveal" data-w-reveal-store-value="grouping" />
<div data-controller="w-reveal" data-w-reveal-store-value="grouping" />
```